src/frontend/js/components/Modal/_styles.scss

```diff
@@ -31,20 +31,12 @@
   }
 
   &__closeButton {
-    @include button-reset-style();
-    background-color: r-theme-val(modal, close-background);
-    @if r-theme-val(modal, close-border) != null {
-      border: thin solid r-theme-val(modal, close-border);
-    }
     box-sizing: initial;
-    color: r-theme-val(modal, close-color);
```

Comment:
If we remove `close-*` variables from `modal` theme, we should remove them from `_theme.scss` then add an entry into the `UPGRADE.md`.